cli/tests/fix_dynamic_import_errors.mjs.out

```diff
@@ -0,0 +1,2 @@
+caught import error from b.mjs
+caught import error from c.mjs
```

Comment:
is the order of these deterministic?

Reply:
The spec for dynamic imports is unclear from [what I am reading](https://tc39.es/proposal-dynamic-import/#sec-hostimportmoduledynamically):

> The actual process performed is implementation defined, but typically consists of performing whatever I/O operations are necessary to allow HostResolveImportedModule to synchronously retrieve the appropriate Module Record, and then calling its Evaluate concrete method. This might require performing similar normalization as HostResolveImportedModule does

Since it isn't required by the spec, I will wildcard them, just so we don't end up with a fragile case that is still technically valid.